tutorial/tests/test_input_output.py

```diff
@@ -14,7 +14,7 @@ def get_data(name: str, data_dir:str="data") -> pl.Path:
     return (pl.Path(current_module.__file__).parent / f"{data_dir}/{name}").resolve()
 
 def reference_solution_find_all_files(f: pl.Path) -> "list[pl.Path]":
-    return list(f.parent.glob(f.name))
+    return list(f.parent.iterdir())
```

Comment:
Also `f.parent.glob("*")` works

Reply:
yes, they do the same. So any solution works.

Reply:
I did, therefore this is fine. We don't want the solution to be identical :)